template.rb

```diff
@@ -243,10 +243,17 @@ def run_rubocop_autocorrections
   run_with_clean_bundler_env "bin/erblint --lint-all -a > /dev/null || true"
 end
 
-def create_database_and_initial_migration
-  return if Dir["db/migrate/**/*.rb"].any?
-  run_with_clean_bundler_env "bin/rails db:create"
-  run_with_clean_bundler_env "bin/rails generate migration initial_migration"
+def add_initial_migrations
+  migration_tasks = %w[
+    action_mailbox:install:migrations
+    action_text:install:migrations
+    active_storage:install
+  ]
```

Comment:
🗒️ Rails 7.1.0 apps will fail to run in CI if  the underlying database tables from frameworks like Active Storage haven't been created yet. To avoid this unfortunate first-run experience, we explicitly run the migrations for these frameworks. This ensures the tables exist.

Reply:
This is now fixed with [this commit](https://github.com/rails/rails/commit/a731dbdfb8940b657946b8febcf7f56d7227beb0) so is no longer required :smile:

Reply:
@deanpcmad that's great news! I'll probably remove this workaround once 7.0.1 is released.